test/NuGet.Core.FuncTests/NuGet.Packaging.FuncTest/SigningTests/SigningUtilityTests.cs

```diff
@@ -141,6 +243,11 @@ internal Test(X509Certificate2 certificate)
                 Options = SigningOptions.CreateFromFilePaths(package.FullName, outputPath, overwrite, signatureProvider, NullLogger.Instance);
             }
 
+            internal string CreateNewTempFile()
```

Comment:
Rename to `CreateNewTempFilePath()` or `GetNewTempFilePath()` since this method does not actually create a new temp file.